src/quo/components/share/share_community_channel_qr_code/style.cljs

```diff
@@ -0,0 +1,23 @@
+(ns quo.components.share.share-community-channel-qr-code.style
+  (:require [quo.foundations.colors :as colors]))
+
+(def outer-container
+  {:border-radius 16
+   :width         "100%"
```

Comment:
does `:flex 1` not work here? 🤔

Reply:
Yeah we had it somewhere in the guidelines about "100%"